crypto/rand/rand_lcl.h

```diff
@@ -10,15 +10,93 @@
 #ifndef HEADER_RAND_LCL_H
 # define HEADER_RAND_LCL_H
 
+# include <openssl/aes.h>
+# include <openssl/evp.h>
+# include <openssl/sha.h>
+# include <openssl/hmac.h>
+# include <openssl/ec.h>
+# include "include/internal/rand.h"
+
 /* we require 256 bits of randomness */
 # define RANDOMNESS_NEEDED (256 / 8)
```

Comment:
It goes all the way back to openssl 0.9.8.   But 128 or 256 is generally believed to be sufficient if the CSPRNG is good quality.  (Ours wasn't, but this PR is a step on the way to making it so)